File: Kaskus/TalkToMe/TalkToMeVb/ConsoleForegroundColor.vb
Friend NotInheritable Class ConsoleForegroundColor
    Implements IDisposable
    Public Sub New(color As ConsoleColor)
        Console.ForegroundColor = color
    End Sub

    Public Sub Dispose() Implements IDisposable.Dispose
        Console.ForegroundColor = ConsoleColor.White
    End Sub
End Class

File: Kaskus/TalkToMe/TalkToMeCs/ConsoleForegroundColor.cs
namespace Dede.App.TalkToMe
{
    using System;

    internal sealed class ConsoleForegroundColor 
        : IDisposable
    {
        public ConsoleForegroundColor(ConsoleColor color)
        {
            Console.ForegroundColor = color;
        }

        public void Dispose()
        {
            Console.ForegroundColor = ConsoleColor.White;
        }
    }
}